client/java/src/main/java/com/linecorp/centraldogma/client/Watcher.java

```diff
@@ -178,4 +182,34 @@ default void watch(Consumer<? super T> listener) {
         requireNonNull(listener, "listener");
         watch((revision, value) -> listener.accept(value));
     }
+
+    /**
+     * @return A new watcher, that reuses the supplied watcher and applies transformation.
```

Comment:
`@return A new watcher, that ...` -> `Returns a new {@link Watcher} that ...`

Reply:
👍

Reply:
Or did you mean that it should not have a `@return` keyword? This was vague https://github.com/line/centraldogma/blob/master/CONTRIBUTING.md#add-javadoc

Reply:
Not really. I just mean that the summary must exist at least. `@return` is a bonus. :-)